### PR TITLE
Trying to build framework

### PR DESCRIPTION
**Pull request by andrewjmeier**
I copied the basic functionality from the framework into a new tabletop directory. Definitely temporary for now, but this allowed me to do the npm publish so the other projects could use it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "tabletop.js",
-  "version": "1.0.0",
+  "name": "tabletop-boardgames",
+  "version": "1.0.2",
```
Comment (qstearns) [inline package.json]: Can we make our version number like .1?

Comment (andrewjmeier) [inline package.json]: It has to be x.x.x can you be more specific? I've already published this so I don't think we can make the version number less

Comment (qstearns) [inline package.json]: I was thinking that might be the case. 0.1.0 was what I was thinking.. Just to indicate this is not a release version

Comment (andrewjmeier) [inline package.json]: Yeah doesn't look like we can do that

Comment (qstearns) [inline package.json]: Okay no problem. Just checking.

**Pull request merged**